cmd/webhooks/main.go

```diff
@@ -81,6 +87,19 @@ func main() {
 		os.Exit(1)
 	}
 	dependentSelector := labels.NewSelector().Add(*dependentRequirement)
+
+	// Setup webhook options
+	disableHTTP2 := func(c *tls.Config) {
+		if enableHTTP2 {
+			return
+		}
+		c.NextProtos = []string{"http/1.1"}
```

Comment:
I hate to ask, is this available as a `const` somewhere?

Reply:
This is a backport of work already merged to main. We want to keep backports as close as possible to the original set of changes, if not 100% identical.

In this case, I don't think a const is necessary; this is the only place this is used. If you wanted to introduce one, we'd need to do it upstream in main, but we most likely wouldn't backport it because it's a no-op change and we want to avoid those in backports.

Reply:
It's actually two places, but I agree with you. I wasn't asking for a change (I did /lgtm), just if it's available as a const. It might be something we want to do upstream/master.